src/lib.rs

```diff
@@ -259,6 +262,14 @@ where
         self
     }
 
+    ///
```

Comment:
Can you provide helpful comments here that describe how to use that feature? (Or at least a link to the rofi documentation where people can lookup the syntax)?